src/Features/Core/Portable/Completion/CompletionItemRules.cs

```diff
@@ -45,20 +45,20 @@ public sealed class CompletionItemRules
         /// <summary>
```

Comment:
Comment is wrong.  but i dont' care that much.  If you do change this, indicate that null means "i don't care", and then things with a higher match priority are picked over those that are lower or null.